### ISSUE TITLE
Overwrite files?

### ISSUE THREAD

**Issue by aholla** [mattstyles/grunt-jsonmin#2]
Hi is it possible to iterate over a folder and its sub folders looking for json files, when it finds it it does the minification and saves over the original one?

This would save having to specify each individual json file.

Something like: 

```
files: [
    {
        expand: true,
         cwd:   '<%= pkg.PATH%>assets/',
        src:    [ '**/*.{json}' ],
        dest:   '<%= pkg.PATH %>assets/',
        ext:    '.json'
    }
]
```

Thanks.

**Comment by mattstyles** [comment]
Yeah, it shouldnt be too much of a change to have it iterate over a file, from memory (which is hazy, its been a while!) that is likely an oversight in the original design.

I'm happy (infact, I'd be delighted) to merge a PR, if not, I'll certainly add it to my todo list, somewhere approaching the top.

**Comment by tastepwalle** [comment]
I know this has been inactive awhile but if someone can suggest another grunt module that accomplishes this sort of behavior with aholla's syntax or similar please ref me and I will create a passing fork for this^^

**Comment by mattstyles** [comment]
I can merge a PR to save you having to fork around an issue

**Comment by tastepwalle** [comment]
Yes sorry that's what I meant..I just need an example to reference before I make a PR. I tried looking at how grunt-copy works but the scope is a bit different..any suggestions?

**Comment by mattstyles** [comment]
It just needs a glob to create the list of files rather than manually specifying them.

I havent used grunt for years but I would be surprised if grunt didnt have a helper function that can accept a glob and spit out a list of matching files. If it does not you just use something like [glob](https://www.npmjs.com/package/glob) to generate a list of files, then iterate over that list (the task already does the iteration).
